### PR TITLE
webapp_using_react_completed

### PR DESCRIPTION
**Pull request by HarsH-singHh**
Screenshot (912)
Screenshot (913)
Screenshot (914)

### CODE REVIEW

**Comment by Sanchitanwar0007** [comment]
Everything looks good and fine. Great work.

File: Assignment-1/code/index.html [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
 <body>
-    <div id="root"></div>
-    <script type="text/babel">
-//React Code here
- ReactDOM.render(<div></div>,document.getElementById('root'));
+  <div id="root"></div>
+  <script type="text/babel">
+    //React Code here
+    //first div
+    var yellowDiv = React.createElement('div', { class: "yellowDiv", style: { display: "flex", 'flex-flow': "row wrap" } }, React.createElement('div', null, null), React.createElement('div', null, null))
```
Comment (Sanchitanwar0007) [inline Assignment-1/code/index.html]: Use className instead of class while writing react codes.

Comment (HarsH-singHh) [reply Assignment-1/code/index.html]: even in an HTML file or is it restricted to a js file only?

Comment (Sanchitanwar0007) [reply Assignment-1/code/index.html]: go and check console its show class error so use className as well in html file because we writing react code here

**Comment by saikiransathpadi** [comment]
layout is correct, use className for css classes and also hyphen separated are not supported in react use camel cased
flex-flow => flexFlow

**Comment by HarsH-singHh** [comment]
> layout is correct, use className for css classes and also hyphen separated are not supported in react use camel cased flex-flow => flexFlow

Handled

**Pull request merged**